### ISSUE TITLE
Update the Support Windows spreadsheet

### ISSUE THREAD

**Issue by kdmccormick** [openedx/build-test-release-wg#199]
The spreadsheet: https://docs.google.com/spreadsheets/d/11DheEtMDGrbA9hsUvZ2SEd4Cc8CaC4mAfoV8SVaLBGI/edit#gid=195838733

The code that generates it: https://github.com/openedx/repo-tools/blob/master/barcalendar.py

The data we should use to update it: https://github.com/overhangio/tutor/blob/master/tutor/templates/config/defaults.yml

## Acceptance Criteria

* Before Olive, update the code & regenerate the sheet based on what Tutor runs now
* Bonus: Make it so the code automatically pulls version info from the Tutor config

**Comment by kdmccormick** [comment]
@BbrSofiane I'll take care of this before the Olive branch cut. Can you create & add an `affects:olive` label?

@nedbat FYI

**Comment by kdmccormick** [comment]
Unfortunately, I don't have time to do this, so I'm unassigning myself.

**Comment by datamatrixnull** [comment]
the python code set up bars for versions of mongodb and the yml doesn't have new info for new version just setup parameters

**Comment by datamatrixnull** [comment]
no, i'm considering fixing this to get an interview for your company

**Comment by kdmccormick** [comment]
@datamatrixnull Sorry about that. I had trouble making sense of your comment, and your account is new, so I assumed wrong. Let us know if there is any way we can help you get started.

**Comment by datamatrixnull** [comment]
okay will look into it

**Comment by MimmyJau** [comment]
I saw there wasn't much activity on this issue, and figured it might not take too long, so I took a first pass at it [here](https://github.com/openedx/repo-tools/pull/414). 

Happy to edit or make any changes.

**Comment by kdmccormick** [comment]
Nice, thanks Mimmy! Once your CLA goes through we'll make sure someone reviews and merges your PR.

**Comment by nedbat** [comment]
The PR is now merged.

**Comment by kdmccormick** [comment]
PR was: https://github.com/openedx/repo-tools/pull/414
